pack.sh

```diff
@@ -40,9 +40,10 @@ scripts/run-rosetta.sh --infuse --pkgs-from $TMPDIR/jsii.txt
 
 # Jsii packaging (all at once using jsii-pacmak)
 echo "Packaging jsii modules" >&2
+rosetta_cache_file=$HOME/.s3buildcache/rosetta-cache.tabl.json
```

Comment:
should this be an environment variable instead? currently I just copied and pasted from run-rosetta and it seems fragile.